src/cheri/zish4add.adoc

```diff
@@ -0,0 +1,39 @@
+[#sh4add_ext]
+== "Zish4add" Extension for 16-byte pointer addition, Version 0.9
+
+ifdef::cheri_standalone_spec[]
+WARNING: This chapter will appear in the unpriv spec (see https://github.com/riscv/riscv-isa-manual/pull/1923).
+endif::[]
+
+The B-extension adds `SH[1|2|3]ADD` instructions, and for RV64 all have an unsigned word .UW version.
```

Comment:
```suggestion
The B-extension adds `SH[1|2|3]ADD` instructions, and for RV64 all have an unsigned word `*.UW` version.
```

Reply:
I took this from https://github.com/riscv/riscv-isa-manual/pull/1923, made the same suggestion there.